src/test/java/de/tum/in/www1/artemis/ParticipationIntegrationTest.java

```diff
@@ -6,13 +6,13 @@
 import java.net.URI;
 import java.time.ZonedDateTime;
 import java.util.*;
+import java.util.stream.Stream;
 
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.CsvSource;
-import org.junit.jupiter.params.provider.EnumSource;
+import org.junit.jupiter.params.provider.*;
```

Comment:
Can we import ``MethodSource`` directly instead of using the star import?

Reply:
Added MethodSource & Argument import instead of star